server/src/test/java/keywhiz/UiAssetsBundleTest.java

```diff
@@ -36,6 +36,14 @@
     client = TestClients.unauthenticatedClient();
   }
 
+  private void checkSecurityHeaders(Response response) {
```

Comment:
I guess I was thinking this would be a test called SecurityHeadersFilterTest just in the same style as UiAssetsBundleTest. That would make it discoverable when changes are made to SecurityHeadersFilter.